kafka2/setup/monit-config.sh

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
```

Comment:
as you have base branch 6413 to this one, please keep code being updated with base branch